package/readme.txt

```diff
@@ -46,6 +46,7 @@ Localization:
 Bug fixes:
 - Fixed copying files to Finder.
 - Fixed drag-and-drop of action buttons in the 'Customize command bar' dialog.
+- Fixed actions not being updated when their shortcuts changed in Preferences. 
```

Comment:
how about being specific here that it's about actions in the command bar and toolbar?